### ISSUE TITLE
questions about training in the paper

### ISSUE THREAD

**Issue by Senwang98** [dong03/MVSS-Net#14]
Hi, @dong03 
Thanks for your nice work!
After reading your paper, I still have some questions:
1. You see 'We re-train FCN (Seg) and MVSS-Net(full setup) from scratch on CASIAv2.', do you pre-train ResFCN first and then use this model to init your MVSS-Net? or just train these two models independently?
2. What's the valid set when training?

**Comment by dong03** [comment]
1. We train models on different datasets independently.
2. A validation set consists of DEFACTO,  on both datasets. (Yet we stopped the training process mainly according to the training loss curve. Better to try some early stopping or cross-validation strategy.
